src/helpers/gsCountFromExtracts.js

```diff
@@ -0,0 +1,9 @@
+export default function gsCountFromExtracts(extracts) {
+  let count = 0;
+  extracts.forEach(extract => {
+    const values = Object.values(extract.data)
```

Comment:
Just to confirm: `extract.data` will always exist?

Reply:
Should prob add an existence check